src/pretix/control/templates/pretixcontrol/event/plugins.html

```diff
@@ -10,20 +10,41 @@ <h1>{% trans "Available plugins" %}</h1>
             software functionality, connect your event to third-party services, or apply other forms of customizations.
         {% endblocktrans %}
     </p>
+    {% if "success" in request.GET %}
+        <div class="alert alert-success">
+            {% trans "Your changes have been saved." %}
+        </div>
+    {% endif %}
+    <p>
+        <div class="row">
+            <div class="col-lg-10">
+                <input type="search" id="plugin_search_input" class="form-control" placeholder="{% trans "Search" %}" autofocus>
+            </div>
+            <div class="col-lg-2">
+                <div class="checkbox">
+                    <label><input type="checkbox" id="plugin_only_active"> {% trans "Show only active plugins" %}</label>
+                </div>
+            </div>
+        </div>
+    </p>
     <form action="" method="post" class="form-horizontal form-plugins">
         {% csrf_token %}
-        {% if "success" in request.GET %}
-            <div class="alert alert-success">
-                {% trans "Your changes have been saved." %}
+        <div id="plugin_search_results" class="panel panel-default collapse">
+            <div class="panel-heading">
+                <button type="button" class="close" aria-label="Close"><span aria-hidden="true">×</span></button>
+                {% trans "Search results" %}
+            </div>
+            <div class="panel-body">
+                <div class="plugin-list"></div>
```

Comment:
We should probably add some CSS to remove the top margin of the first result:

![image](https://github.com/user-attachments/assets/f2f7859e-6813-4864-a779-667bc7d92ff7)